publisher1.go

```diff
@@ -103,21 +103,30 @@ func Publishv1(input chan []*FileEvent,
 			}
 
 			// read ack
-			response := make([]byte, 0, 6)
-			ackbytes := 0
-			for ackbytes != 6 {
-				n, err := socket.Read(response[len(response):cap(response)])
-				if err != nil {
-					emit("Read error looking for ack: %s\n", err)
-					socket.Close()
-					socket = connect(config)
-					continue SendPayload // retry sending on new connection
-				} else {
-					ackbytes += n
-				}
-			}
+			response := make([]byte, 6)
```

Comment:
Can you change your indentation to use tabs instead of spaces? You can try automating this with `gofmt -w publisher1.go` which will restyle the whole file.